apps/stirling-pdf/0.36.3/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.36.3
+    image: frooodle/s-pdf:0.36.4
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There is one difference in the Dockerfile between lines 2 and 3 in both versions.

In version 1, it uses `frooodle/s-pdf:0.36.3` as the image tag.
In version 2, it changes to `frooodle/s-pdf:0.36.4`.

This change only affects the docker image being used by the container named "stirling-pdf". The container name and restart settings remain unchanged, indicating that no critical configuration has been altered.

If you need to update an existing service running this container based on version 2 tags, you'll have to pull the new Docker image and restart the relevant containers using your current container management mechanism (e.g., Kubernetes, Docker Compose).